INSTALL.md

```diff
@@ -19,31 +19,26 @@ Requirements
 
 ### Mandatory
 
-- **gcc** 4.6 to 5.X (depends on your current [CUDA version](https://gist.github.com/ax3l/9489132))
+- **gcc** 4.8 to 5.X (depends on your current [CUDA version](https://gist.github.com/ax3l/9489132))
   - *Debian/Ubuntu:*
     - `sudo apt-get install gcc-4.9 g++-4.9 build-essential`
     - `sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.9 60 --slave /usr/bin/g++ g++ /usr/bin/g++-4.9`
   - *Arch Linux:*
     - `sudo pacman --sync base-devel`
     - the installed version of **gcc** might be too new. [Compile an older gcc](https://gist.github.com/slizzered/a9dc4e13cb1c7fffec53)
-  - *experimental alternatives:* **icc** since **CUDA 5.5**
 
-- C++98 build: [CUDA 5.0-7.0](https://developer.nvidia.com/cuda-downloads)
-  - **Attention:** You must use at least the 5.5+ [drivers](http://www.nvidia.com/Drivers)
-    even if you run with CUDA 5.0. Supported drivers: 319.82+/331.22+
+- C++98 build: [CUDA 5.5-7.0](https://developer.nvidia.com/cuda-downloads)
   - *Arch Linux:* `sudo pacman --sync cuda`
 - C++11 build: [CUDA 7.5+](https://developer.nvidia.com/cuda-downloads)
-  - `export PIC_COMPILE_SUITE_CMAKE="-DCMAKE_CXX_STANDARD=11"`
-  - `export CPPFLAGS="-std=c++11"`
-  - if you run `$PICSRC/configure` below, add `-c"-DCMAKE_CXX_STANDARD=11"` to it
+  - note: we recommend using **gcc 4.9**
   - *Arch Linux:* `sudo pacman --sync cuda`
 
 - at least one **CUDA** capable **GPU**
   - *Compute capability* **sm\_20** or higher
```

Comment:
Could we add a note that we mark sm_20 as deprecated? In CUDA 8 sm_20 is also deprecated.